packages/types/src/types/OVirtStorageDomains.ts

```diff
@@ -0,0 +1,13 @@
+/** Represents an oVirt storage domain. */
+export interface OVirtStorageDomains {
```

Comment:
@yaacov  @sjd78 
the plural in the name is not intuitive for me - is it some convention?
Applies to other files/types.

Reply:
no convention, I'm open to better names

Reply:
ok. In this place I would just use `OVirtStorageDomain` (the singular form).
The plural form is justified only if the object behind is a wrapper around some collection which cannot be expressed via simple array .